alws/crud/errata.py

```diff
@@ -540,17 +541,24 @@ async def get_matching_albs_packages(
         errata_package.source_srpm = src_nevra.name
         items_to_insert.append(mapping)
         errata_package.albs_packages.append(mapping)
-        return items_to_insert
+        package_type["type"] = "prod"
```

Comment:
Not critical, just a personal taste, I'd prefer to use an enum here rather than checking on either `prod` or `build` strings later on.